mix.exs

```diff
@@ -46,21 +48,36 @@ defmodule Demo.MixProject do
   # Specifies your project dependencies.
   #
   # Type `mix help deps` for examples and options.
-  defp deps do
+  # This refers to dependencies shared among :prod, :test_local, :test, and :dev environments.
+  defp deps() do
     [
-      {:credo, "~> 1.4", only: [:dev, :test], runtime: false},
-      {:dialyxir, "~> 1.0", only: [:dev, :test], runtime: false},
-      {:excoveralls, "~> 0.11", only: :test},
+      {:credo, "~> 1.4", only: [:dev, :test, :test_local], runtime: false},
+      {:dialyxir, "~> 1.0", only: [:dev, :test, :test_local], runtime: false},
+      {:excoveralls, "~> 0.11", only: [:test, :test_local]},
       {:phoenix, "~> 1.5.4"},
       {:phoenix_live_view, "~> 0.15"},
-      {:floki, ">= 0.0.0", only: :test},
+      {:floki, ">= 0.0.0", only: [:test, :test_local]},
       {:phoenix_html, "~> 2.11"},
       {:phoenix_live_reload, "~> 1.2", only: :dev},
       {:telemetry_metrics, "~> 0.4"},
       {:telemetry_poller, "~> 0.4"},
       {:gettext, "~> 0.11"},
       {:jason, "~> 1.0"},
-      {:plug_cowboy, "~> 2.0"},
+      {:plug_cowboy, "~> 2.0"}
+    ] ++ uncharted_deps(local: File.exists?(@uncharted_path))
+  end
+
+  # This refers to dependencies only needed for local development purposes.
+  defp uncharted_deps(local: true) do
+    [
+      {:uncharted, only: [:dev, :test], path: "../uncharted/uncharted"},
+      {:uncharted_phoenix, only: [:dev, :test], path: "../uncharted/uncharted_phoenix"}
```

Comment:
Did you want to use `@unchared_path` here as well? (in which case you can modify the statement in the Readme)